src/audio/src/src.c

```diff
@@ -946,16 +946,14 @@ static int src_check_buffer_sizes(struct comp_data *cd,
 
 	n = audio_stream_frame_bytes(source_stream) * (blk_in + cd->source_frames);
```

Comment:
minor nit: In git commit descriptions, I'd avoid acronyms like "PR". These are recorded for the history books, so better this way. Also motivation of the change is good to recap in git commit. Not so important in simple patches like this, but still doesn't hurt.